<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Launcher browser]]>
    </line>
  </reporter-output>
  <suite started-at="2022-07-10T17:28:50 IST" name="Default suite" finished-at="2022-07-10T17:29:44 IST" duration-ms="54351">
    <groups>
    </groups>
    <test started-at="2022-07-10T17:28:50 IST" name="Default test" finished-at="2022-07-10T17:29:44 IST" duration-ms="54351">
      <class name="KiteTest.validateuserIDforPropertyfile">
        <test-method is-config="true" signature="BrowserLaunch()[pri:0, instance:KiteTest.validateuserIDforPropertyfile@d9345cd]" started-at="2022-07-10T17:28:50 IST" name="BrowserLaunch" finished-at="2022-07-10T17:29:39 IST" duration-ms="48978" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Launcher browser]]>
            </line>
          </reporter-output>
        </test-method> <!-- BrowserLaunch -->
        <test-method is-config="true" signature="logintoapp()[pri:0, instance:KiteTest.validateuserIDforPropertyfile@d9345cd]" started-at="2022-07-10T17:29:39 IST" name="logintoapp" finished-at="2022-07-10T17:29:40 IST" duration-ms="1127" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintoapp -->
        <test-method signature="validateuserID()[pri:0, instance:KiteTest.validateuserIDforPropertyfile@d9345cd]" started-at="2022-07-10T17:29:40 IST" name="validateuserID" finished-at="2022-07-10T17:29:44 IST" duration-ms="3856" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateuserID -->
        <test-method is-config="true" signature="logout()[pri:0, instance:KiteTest.validateuserIDforPropertyfile@d9345cd]" started-at="2022-07-10T17:29:44 IST" name="logout" finished-at="2022-07-10T17:29:44 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at KiteTest.validateuserIDforPropertyfile.logout(validateuserIDforPropertyfile.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="Browserclose()[pri:0, instance:KiteTest.validateuserIDforPropertyfile@d9345cd]" started-at="2022-07-10T17:29:44 IST" name="Browserclose" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1657454384652" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Browserclose -->
      </class> <!-- KiteTest.validateuserIDforPropertyfile -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
